lms/static/sass/views/_verification.scss

```diff
@@ -679,21 +679,22 @@
 
       .task {
         @extend %ui-window;
-        float: left;
+        @include float(left);
+        @include margin-right(flex-gutter());
         width: flex-grid(6,12);
-        margin-right: flex-gutter();
       }
 
       .controls {
         padding: ($baseline*0.75) $baseline;
         background: $m-gray-l4;
+        text-align: center;
 
         .list-controls {
           position: relative;
         }
 
         .control {
-          position: absolute;
+          // position: absolute;
```

Comment:
If this isn't needed anymore, you can just remove it instead of commenting it out.